test/androidTest/java/org/thoughtcrime/securesms/util/ConversationActions.java

```diff
@@ -69,141 +53,158 @@ public void conversationListActivityTest() {
                                                 0)),
                                 0),
                         isDisplayed()));
-        appCompatEditText3.perform(replaceText("5144022093"), closeSoftKeyboard());
-
-        try {
-            Thread.sleep(1000);
-        } catch (InterruptedException e) {
-            e.printStackTrace();
-        }
+        appCompatEditText.perform(replaceText(number), closeSoftKeyboard());
 
         onView(withId(R.id.recycler_view))
                 .perform(RecyclerViewActions.actionOnItemAtPosition(0, click()));
+    }
 
-        try {
-            Thread.sleep(1000);
-        } catch (InterruptedException e) {
-            e.printStackTrace();
-        }
+    public static void goToConversation(String number) {
```

Comment:
just curious what does the number represent? is it even used?

Reply:
Good point, will make it use the number as was the intention.